climada/util/coordinates.py

```diff
@@ -890,11 +890,17 @@ def assign_coordinates(coords, coords_to_assign, method="NN", distance="haversin
         Index into `coords_to_assign`. Note that the value `-1` is used to indicate that no
         matching coordinate has been found, even though `-1` is a valid index in NumPy!
     """
+
+    if not coords.any() or not coords_to_assign.any():
+        return np.array([])
```

Comment:
If `coords` is empty, I'd expect to return an arrray the size of `coords_to_assign` with all values set to -1

Reply:
Done here: 
59a0894